docs/tutorial/appendix/source-JSON-YAML/index.md

```diff
@@ -0,0 +1,274 @@
+---
+title: "Sourcing from JSON or YAML"
+---
+
+# Beforehand
+
+As you're presented with Gatsby, you'll see that you have at your disposal a myriad of ways to pull data from into your website, ranging from the most used ones like [Contenfull](https://www.contentful.com/), or [WordPress](https://wordpress.com/), [Drupal](https://www.drupal.org/), to the most edgier ones like [ButterCMS](https://buttercms.com/), [GraphCMS](https://graphcms.com/) and their appropriate plugins to ensure connectivity.
+
+With this tutorial, it will help you return to the origins and instead of pulling data in from the providers above, you'll use a plain JSON file and YAML to achieve this.
+
+It will presented a way to use either JSON or YAML files, for creating the bare structure of a gatsby website, by adding some pages dynamically with either one of the above.
+
+Also a way to load contents from both sources to a page.
```

Comment:
This sentence could be combined with the paragraph above:

"You will encounter multiple ways to use JSON or YAML files for creating the bare structure of a Gatsby website, by adding some pages with content pulled dynamically from either source."